functions/raddStoreRegistryLambda/src/app/csvUtils.js

```diff
@@ -97,9 +98,62 @@ function sanitizeCSVField(field) {
   return `"${escapedField}"`;
 }
 
+/**
+ * Check if the AWS address is valid based on scores and thresholds
+ *
+ * @param {Object} scores - AWS address scores object (subRegion, locality, postalCode, addressNumber, overall)
+ * @returns {boolean} True if address is valid, false otherwise
+ */
+function isAWSAddressValid(scores) {
+  if (!scores) return false;
+
+  const requiredScores = [
+    'subRegion',
+    'locality',
+    'postalCode',
+    'addressNumber',
+  ];
+
+  const thresholds = {
+    subRegion: parseFloat(process.env.SUBREGION_THRESHOLD),
+    locality: parseFloat(process.env.LOCALITY_THRESHOLD),
+    postalCode: parseFloat(process.env.POSTALCODE_THRESHOLD),
+    addressNumber: parseFloat(process.env.ADDRESSNUMBER_THRESHOLD),
+    overall: parseFloat(process.env.OVERALL_THRESHOLD),
+  };
+
+  // Check all required fields exist and are truthy
+  const hasAllRequiredFields = requiredScores.every(
+    (key) => scores[key] != null && scores[key] !== false
+  );
+
+  if (!hasAllRequiredFields || scores.overall == null) {
+    return false;
+  }
+
+  // Check if all required scores and overall are equal to 1
```

Comment:
The thresholds variables are required? If yes instead of doing this check and the next ones, we can simply check that all the scores are >= the thresholds defined in the configuration